website/src/documentation/transforms/python/element-wise/tostring.md

```diff
@@ -19,9 +19,38 @@ limitations under the License.
 -->
 
 # ToString
+
+<script type="text/javascript">
+localStorage.setItem('language', 'language-py')
+</script>
+
 Transforms every element in an input collection a string.
```

Comment:
I spot a typo I made:

"collection a string"->"collection to a string"

Reply:
Fixed